app/models/points_manager.rb

```diff
@@ -0,0 +1,21 @@
+class PointsManager
+  def initialize(current_user, habit)
+    @user = current_user
+    @habit = habit
+  end
+
+  def add_points
+    habit_count = Dailyhabit.where(
+      user_id: @user.id).where(
+      'date >= ?', DateTime.now.beginning_of_day..DateTime.now.end_of_day
```

Comment:
Indent the first parameter one step more than the previous line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.